docs/package.json

```diff
@@ -19,6 +19,7 @@
     "format": "remark --use \"remark-mdx=printWidth:80\" docs/**/*.mdx -o"
   },
   "dependencies": {
+    "@databricks/design-system": "^1.12.17",
```

Comment:
Does databricks/design-system good docs?

Reply:
I'm curious why we chose databricks/design-system.

Reply:
> we will avoid adding libraries for the moment

Sounds good! To be clear, I'm not opposed to using a library like `databricks/design-system` as long as it works and has good docs.